paymentapi/src/main/java/fi/hel/verkkokauppa/payment/paytrail/context/PaytrailPaymentContextBuilder.java

```diff
@@ -24,16 +28,17 @@ public class PaytrailPaymentContextBuilder {
     private CommonServiceConfigurationClient commonServiceConfigurationClient;
 
 
-    public PaytrailPaymentContext buildFor(String namespace, String merchantId) {
+    public PaytrailPaymentContext buildFor(String namespace, String merchantId, boolean useShopInShop) {
```

Comment:
Added new boolean flag that enables the creation of PytrailPaymentContext for shop-in-shop and normal merchant flow.

Reply:
Add this as comment to function, we can remember it better when we stumble these codes later